code/modules/SCPmodular/scps/euclid/SCP-294.dm

```diff
@@ -0,0 +1,72 @@
+GLOBAL_LIST_EMPTY(scp294_reagents)
+
+/obj/machinery/scp294
+	name = "SCP-294"
+	desc = "A standard coffee vending machine."
+	icon = 'code/modules/SCPmodular/spcicon/scpobj/scp294.dmi'
+	icon_state = "coffee_294"
+	layer = 2.9
+	anchored = 1
+	density = 1
+	var/uses_left = 12
+	var/last_use = 0
+	var/restocking_timer = 0
+
+/obj/machinery/scp294/New()
```

Comment:
I'm not entirely sure but shouldn't this be Initialize()? I'll look into this later.